### PR TITLE
My Jetpack: Version 1.0

### PR DESCRIPTION
**Pull request by kraftbj**
Closes #23690

#### Changes proposed in this Pull Request:

* Releases My Jetpack 1.0.0

#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No
#### Testing instructions:

* CI Passes

### CODE REVIEW

File: projects/packages/my-jetpack/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.0.0 - 2022-03-30
```
Comment (anomiex) [inline projects/packages/my-jetpack/CHANGELOG.md:8]:
```suggestion
## [1.0.0] - 2022-03-30
```

File: projects/packages/my-jetpack/CHANGELOG.md [hunk 2]
```diff
@@ -400,6 +409,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Created package
 
+[0.6.13]: https://github.com/Automattic/jetpack-my-jetpack/compare/0.6.12...0.6.13
```
Comment (anomiex) [inline projects/packages/my-jetpack/CHANGELOG.md:412]:
```suggestion
[1.0.0]: https://github.com/Automattic/jetpack-my-jetpack/compare/0.6.12...1.0.0
```

Comment (kraftbj) [reply projects/packages/my-jetpack/CHANGELOG.md:412]: Ah we should make the "linked version number" error more descriptive. It yelled at me that the version wasn't linked to anything, but I didn't look further down the doc.

**Comment by samiff** [comment]
I've added the DNM merge label for now after some internal Slack chatter: p1648660346785599-slack-C01U2KGS2PQ

**Comment by samiff** [comment]
After some [Slack](p1648660346785599-slack-C01U2KGS2PQ) chatter, we decided it'd be best to hold off on releasing MJ 1.0 until after Jetpack 10.8 ships. We can still cherry-pick any necessary changes for the package to the release branch 👍 I'd be happy to help with MJ 1.0 after 10.8 ships if needed.

**Pull request closed without merging**